src/features/soulkeys/BuyOrSellSoulKeys.tsx

```diff
@@ -120,6 +121,7 @@ export const BuyOrSellSoulKeys = ({
         setProgress('Done!');
         refresh();
         await syncKeys();
+        queryClient.invalidateQueries(['soulKeys_hasOwnAndOtherKeys']);
```

Comment:
I don't see this key being used anywhere... which is a good reason to use a CONSTANT so that its clearer with type errors if its the smae name or not. @wingfeatherwave

Reply:
fixed it.